pcsx2/IPU/IPU.cpp

```diff
@@ -243,6 +243,10 @@ __fi u64 ipuRead64(u32 mem)
 		ipucase(IPU_CMD): // IPU_CMD
 			if (ipuRegs.cmd.DATA & 0xffffff)
 				IPU_LOG("read64: IPU_CMD=BUSY=%x, DATA=%08X", ipuRegs.cmd.BUSY ? 1 : 0, ipuRegs.cmd.DATA);
+			if (ipuRegs.cmd.BUSY && ipuRegs.cmd.DATA == 0x000001B7) { //prevents Enthusia - Professional Racing end of video crash.
```

Comment:
Slightly better, but IMO still not enough. What do you mean by "signals"? is it a command? is it a value which the game just happens to use? and most importantly, I'm assuming something makes you think that it won't affect other games negatively, then this info is missing too.